python_modules/dagster/dagster/core/host_representation/origin.py

```diff
@@ -197,7 +197,13 @@ def create_location(self) -> NoReturn:
 
     @contextmanager
     def create_test_location(self):
-        from dagster.core.workspace.dynamic_workspace import DynamicWorkspace
+        """Back-compat after create_test_location was renamed to create_single_location"""
+        with self.create_single_location() as location:
+            yield location
+
+    @contextmanager
+    def create_single_location(self):
```

Comment:
bit confused about this - maybe worth some more exposition in the comment or double check it makes sense